index.ts

```diff
@@ -506,8 +507,10 @@ const getRepoPath = (url: URL | HTMLAnchorElement | Location = location): string
 	return undefined;
 };
 
-/** Get the 'user/repo' part from an URL. Tries using the canonical URL to avoid capitalization errors in the `location` URL */
-const getRepoURL = (url?: URL | HTMLAnchorElement | Location): string => {
+/** Get the 'user/repo' part from an URL. Tries using the canonical URL to avoid capitalization errors in the `location` URL
+@deprecated Use getRepositoryInfo().url
+*/
+const getRepoURL = (url?: URL | Location): string => {
```

Comment:
Unlike `getRepoPath`, `getRepoURL` did not actually check `isRepoURL`, so I need to ensure that `getRepoURL` wasn't used improperly in RGH